product.html

```diff
@@ -141,5 +142,154 @@
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_2923_446">
+                      <rect width="16" height="16" fill="white"/>
+                    </clipPath>
+                  </defs>
+              </svg> Retiro en Tienda</button>
+            </div>
+          </div>
+          <div class="product__add-card">
+            <div class="product__add-card-counter">
+              <button class="product__add-card-counter-button product__add-card-counter-button--more-add">
+                <svg xmlns="http://www.w3.org/2000/svg" width="26" height="30" viewBox="0 0 26 30" fill="none">
+                <path d="M5.9574 15H20.4539" stroke="#D8D8D8" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                </svg>
+              </button>             
+              <p class="product__add-card-number">1</p>
```

Comment:
Esto deberia ser un input para permitir que el visitante pueda cambiar la cantidad a su antojo
```suggestion
              <input type="number" name="quantity" class="product__add-card-number" value="1">
```